OrbitGl/TimeGraph.cpp

```diff
@@ -699,6 +676,13 @@ void TimeGraph::SortTracks() {
       sortedThreadIds = filteredThreadIds;
     }
 
+    sorted_tracks_.emplace_back(GetOrCreateThreadTrack(0));
```

Comment:
I don't think it matters because of the way the SchedulerTrack is rendered, but why is the SchedulerTrack put at the end of the of the sorted_tracks_?  We might want to make it the first track if we change the rendering of the SchedulerTrack to be relative and not absolute.

Reply:
It's actually put at the beginning. Above, the thread ids are sorted in a different vector first, then inserted here into the sorted_tracks_ vector. I made it more obvious by clearing the sorted_tracks_ vector here right before inserting.